spec/rubocop/cop/rspec/cop_spec.rb

```diff
@@ -21,13 +21,14 @@
 
   let(:fake_cop) do
     stub_const('RuboCop::RSpec', Module.new)
-    # rubocop:disable ClassAndModuleChildren, RSpec/LeakyConstantDeclaration
```

Comment:
I think `Leaky` is still detected here.

Reply:
I have changed this twice and my editor keeps autocorrecting it away (incorrectly) 😆